launcher/cog-launcher.c

```diff
@@ -987,6 +988,16 @@ cog_launcher_add_web_settings_option_entries(CogLauncher *launcher)
     g_application_add_option_group(G_APPLICATION(launcher), g_steal_pointer(&option_group));
 }
 
+static gboolean
+option_entry_parse_gamepad(const char *option_name, const char *value, void *data, GError **error)
+{
+    if (!cog_gamepad_parse_backend(value, error))
+        return FALSE;
+
+    s_options.gamepad = g_strdup(value);
```

Comment:
This could call `cog_gamepad_set_backend()` directly here, no need to save the passed value.

Reply:
done

Reply:
I kept both methods, for users, perhaps they don't want to parse arguments, but only set the gamepad backend.